capabilities.json

```diff
@@ -119,5 +119,15 @@
                 }
             }
         }
+    },
+    "sorting": {
+        "implicit": {
+            "clauses": [
+                {
+                    "role": "Weight",
```

Comment:
If you do not pre-sort the data, you can get them in a different order for different browsers